general-remarks.md

```diff
@@ -17,11 +17,11 @@
 
 The documentation uses term **Property** to describe a client of channel manager or PMS and **Space Type** to refer to a space sold/offered by the **Property**. In hospitality world, a **Property** is a hotel \(hostel, etc.\) and **Space Type** is a room \(bed, etc.\). But **Property** could also \(for example\) be some sport facility and then the **Space Type** would refer to a squash court, that is why those unusual, yet general, terms.
 
-There can be multiple connections between one property and one channel manager. All operations of both API sides require `clientToken` to be present in the request, On Mews API side, the `clientToken` determines a concrete channel manager, so it is same for all connections. The `clientToken` on the [Channel Manager API](channel-manager-api.md) side can be same or different, it is up to an agreement, as long as the `clientToken` is unique for Mews and same for all Mews connections. All operations regarding an established connection with property require `connectionToken` as well, `connectionToken` determines a concrete connection between the channel manager and a Mews hotel, it is unique for each connection.
+There can be multiple connections between one property and one channel manager. All operations of both API sides require `clientToken` to be present in the request, On Mews API side, the `clientToken` determines a concrete channel manager, so it is same for all connections. The `clientToken` on the [Channel Manager API](channel-manager-api.md) side can be same or different, it is up to an agreement, as long as the `clientToken` is unique for Mews and same for all Mews connections. All operations regarding an established connection with property require `connectionToken` as well, `connectionToken` determines a concrete connection between the channel manager and a Mews property, it is unique for each connection.
 
 If you are interested in changes and updates of this API, check [Changelog](changelog.md#changelog).
 
-If you are interested in the list of Channels \(i.e. OTAs\), check [Channels](channels.md#channels). If you are connected to some Channels that are not in the list, please provide them to us, we will add them with some code.
+If you are interested in the list of Channels \(i.e. OTAs\), check [Channels](channels.md#channels). If you are connected to some Channels that are not in the list, please email them to channel-api@mews.com, we will add them with some code.
```

Comment:
Do we want to have hardcoded email on more places ? So once we change it you will have to change it everywhere and dont forgot about it

Reply:
From the users perspective, I think we should go with single email, there is already integrations@mews.com somewhere in the beginning of the docs, no?

Reply:
There was one more PR to change it to this one.

Reply:
I agree that it requires maintenance to update email address in case of changes but without it partners can't reach us and usually end up contacting our Support team. Of course they can use the chat bot but I would say this would not be great experience since we ask them for URLs to add them to Mews.
As per having different emails (integrations@ and channel@), it was agreed with Jirka that these two are different APIs and have completely different workflows (building, testing, zap automation). So having two makes it easier for us from operational perspective.

Reply:
Ok, so we have connector@ for the other API? That would make sense to me. integrations@ is superset of channel, connector and other integrations.

Reply:
This is a fair point but we dont. Since most of partners use Connector we just agreed to keep integrations@ (which is btw marketplace@, but it gets routed) for Connector questions. I will raise this with Josef.

Reply:
Yeah, if it makes sense to have dedicated email for CHM, it would make sense to me to have dedicated email for connector so that connector requests can be routed directly to connector TPSM. And we can still keep integration@ for general enquiries / messages that get triaged and assigned to one of the two "subteams".

Reply:
Spoke to Josef. The plan is to keep these two emails (marketplace@ and channel@) and not to create extra ones because it will require to change all the automation behind it and we will need to kill this connector@ and channel@ emails once we merge the APIs. In addition, we are going thru team restructuring, so every team member will become an expert in both APIs. So we are moving towards having one email, which will be marketplace@ and have unified workflows automation and Intercom queue.

Reply:
👌